### DISCUSSION TITLE
有没有考虑增加更多字符集单字支持？

### DISCUSSION THREAD

**Discussion by elisir516** [KyleBing/rime-wubi86-jidian#104] [Ideas]
比如将黄狗五笔超大字符集版中单字部分提取出来，然后与原码表对比去重后，添加到一个新的码表中，在原码表中搜索不中时，再搜索这个超大字符集码表？

**Comment by KyleBing** [comment]
其实该库主要目标是用于日常输入，并不是特殊字符的输入。我个人一直感觉像那些古怪的字都不应该再使用了，所以不会对字符集进行扩展。
